README.md

```diff
@@ -232,8 +232,10 @@ can be used as:
 
 Prerequisites:
 
-- The environment variable `CASC_VAULT_PW` must be present, if token is not used. (Vault password.)
-- The environment variable `CASC_VAULT_USER` must be present, if token is not used. (Vault username.)
+- The environment variable `CASC_VAULT_PW` must be present, if token is not used and appRole/Secret is not used. (Vault password.)
+- The environment variable `CASC_VAULT_USER` must be present, if token is not used and appRole/Secret is not used. (Vault username.)
+- The environment variable `CASC_VAULT_APPROLE` must be present, if token is not used and U/P not used. (Vault AppRole ID.)
+- The environment variable `CASC_VAULT_APPROLE_SECRET` must be present, it token is not used and U/P not used. (Value AppRole Secret ID.)
```

Comment:
even more environment variables 🤭 I don't know if there is a better solution.

Reply:
I suppose to reduce the number of environment variables, you could re-use CASC_VAULT_USER and CASC_VAULT_PW, and if CASC_VAULT_MOUNT == 'approle' then use the loginByAppRole() method instead of loginByUserPass() but that might make it less transparent and harder to document clearly how the variables should be used.
Happy to take a steer from you though if you would prefer this approach.

Reply:
That would make it even more confusing 😅 I guess the current approach is good enough.